web/src/routes/MyPages/AuthListPage.tsx

```diff
@@ -1,5 +1,279 @@
+`;
+
+const AuthListContainer = styled.div`
+  padding: 5vw;
+  margin-left: calc(10vh - 5vmin);
+  margin-right: calc(10vh - 5vmin);
+  margin-top: calc(3vw - 2vmin);
+  min-width: 40vh;
+  height: 80vh;
+  background-color: ${(props) => props.theme.colors.container};
+`;
+
+const ContainerBox = styled.button`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: left;
+  width: fit-content;
+  margin-bottom: 1vh;
+  border: none;
+  border-radius: 30px;
+  cursor: pointer;
+  background-color: ${(props) => props.theme.colors.container};
+
+  &:hover {
+    background-color: ${(props) => (!props.disabled ? props.theme.colors.background : props.theme.colors.container)};
+    cursor: ${(props) => (!props.disabled ? 'pointer' : 'auto')};
+  }
+`;
+
+const Form = styled.form`
+  display: flex;
+  flex-direction: column;
+`;
+
+const AuthListTitle = styled.span`
+  text-align: left;
+  font-size: calc(2vh + 2vmin);
+  font-weight: 900;
+  margin-bottom: 2vh;
+  color: ${(props) => props.theme.colors.default};
+  cursor: pointer;
+`;
+
+const BeforeText = styled.span`
+  text-align: left;
+  font-size: calc(1vh + 1.4vmin);
+  font-weight: 900;
+  min-width: 80px;
+  max-width: 40vw;
+  width: fit-content;
+  color: ${(props) => props.theme.colors.default};
+`;
+
+const AfterText = styled.span`
+  text-align: left;
+  font-size: calc(1vh + 1.4vmin);
+  font-weight: 900;
+  min-width: 80px;
+  max-width: 40vw;
+  width: fit-content;
+  color: ${(props) => props.theme.colors.btnhover};
+`;
+
+type State = {
+  kakao: boolean;
+  naver: boolean;
+  google: boolean;
+  identityCard: boolean;
+  driverCard: boolean;
+  schoolMail: boolean;
+};
+
+type Action =
+  | { type: 'AUTH_KAKAO'; payload: boolean }
+  | { type: 'AUTH_NAVER'; payload: boolean }
+  | { type: 'AUTH_GOOGLE'; payload: boolean }
+  | { type: 'AUTH_IDENTITY'; payload: boolean }
+  | { type: 'AUTH_DRIVER'; payload: boolean }
+  | { type: 'AUTH_SCHOOLMAIL'; payload: boolean };
+
+// test
+const initialState = {
+  kakao: true,
+  naver: false,
+  google: true,
+  identityCard: false,
+  driverCard: false,
+  schoolMail: false,
+};
+
+const reducer = (state: State, action: Action): State => {
+  switch (action.type) {
+    case 'AUTH_KAKAO':
+      return { ...state, kakao: action.payload };
+    case 'AUTH_NAVER':
+      return { ...state, naver: action.payload };
+    case 'AUTH_GOOGLE':
+      return { ...state, google: action.payload };
+    case 'AUTH_IDENTITY':
+      return { ...state, identityCard: action.payload };
+    case 'AUTH_DRIVER':
+      return { ...state, driverCard: action.payload };
+    case 'AUTH_SCHOOLMAIL':
+      return { ...state, schoolMail: action.payload };
+    default:
+      return state;
+  }
+};
+
+export default function MyPage() {
+  const [theme, toggleTheme] = useTheme();
+  const navigate = useNavigate();
+
+  const [state, dispatch] = useReducer(reducer, initialState);
+
+  // Input Data list
```

Comment:
Please remove this comment. Or make your comment in a JSDoc style so that it's easily recognizable